Shapes/index.js

```diff
@@ -0,0 +1,35 @@
+
+/*   Area Of Circle   */
```

Comment:
You are using descriptive names for the variables and functions, so comments like these aren't needed.